src/atomdb/AtomDB.h

```diff
@@ -24,13 +24,12 @@ class AtomDB : public HandleDecoder {
 
     virtual shared_ptr<atomdb_api_types::HandleSet> query_for_pattern(
         const LinkTemplateInterface& link_template) = 0;
-    virtual shared_ptr<atomdb_api_types::HandleList> query_for_targets(shared_ptr<char> link_handle) = 0;
```

Comment:
@andre-senna heads up that I removed this and I'm not sure if this would decrease performance, what do you think?

Reply:
I think it's OK